Makefile

```diff
@@ -294,7 +294,10 @@ PKGS_riscv64=pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/grub     \
              pkg/mkimage-raw-efi pkg/uefi pkg/u-boot pkg/grub pkg/new-kernel \
 	     pkg/debug pkg/dom0-ztools pkg/gpt-tools pkg/storage-init pkg/mkrootfs-squash
 # alpine-base and alpine must be the first packages to build
-PKGS=pkg/alpine-base pkg/alpine $(PKGS_$(ZARCH))
+PKGS=pkg/alpine $(PKGS_$(ZARCH))
+# eve-alpine-base is bootstrap image for eve-alpine
+# if you want to bootstrap eve-alpine again, uncomment the line below
+# PKGS:=pkg/alpine-base $(PKGS)
```

Comment:
Would it make sense to add a comment with a reference to the markdown file here?